### ISSUE TITLE
Fail to import due to a "DLL load failed error"

### ISSUE THREAD

**Issue by jgv115** [r9y9/pylibfreenect2#44]
I'm running Windows 10. Protonect works fine and I have Microsoft Visual C++ 2015 Redistributable (x64) and (x86) installed.

When I try to import pylibfreenect2, it comes up with the error:

```
from .libfreenect2 import *
ImportError: DLL load failed: The specified module could not be found.
```

**Comment by r9y9** [comment]
Looks like there are missing DLLs not in `PATH` and it's not specific problem to pylibfreenect2. While googling I found there are many similar reports. e.g., https://github.com/skvark/opencv-python/issues/36.

**Comment by hjw1007** [comment]
I also meet this error, I want to know that how to deal with it at last, thanks

    from pylibfreenect2 import Freenect2, SyncMultiFrameListener
ImportError: cannot import name 'Freenect2'

**Comment by adarvit** [comment]
i have the same problem
@jgv115, @hjw1007  did you solve it?

**Comment by jmarkow** [comment]
Hi all, I ran into the same problem and was able to fix it through copying all of dlls generated in the libfreenect2 build (in `install/bin` by default in the libfreenect2 source directory),

```
freenect2.dll
glfw3.dll
libusb-1.0.dll
turbojpeg.dll
```

to my miniconda3 path,

`C:\Users\<username>\AppData\Local\Continuum\miniconda3`

Alternatively I believe you could append the path to the dlls to your path variable,

```
set path=%PATH%;C:\libfreenect2\install\bin
```

**Comment by r9y9** [comment]
> Alternatively I believe you could append the path to the dlls to your path variable,

Yes. See also: https://github.com/r9y9/pylibfreenect2/blob/78b8748ea356d8ed61d862e24e8eab934d94d6e3/appveyor.yml#L36-L41.

**Comment by yzf123456** [comment]
https://github.com/jmarkow
 i just  append the path to the dlls to your path variable，but it dose not work ,so what should i do? and i do not know  where to copy all of dlls generated in the libfreenect2 build  in my computer? Thank you!

**Comment by xieshenru** [comment]
> C:\Users\<username>\AppData\Local\Continuum\miniconda3

I cannot do "from pylibfreenect2 import Freenect2", where is miniconda3 path"C:\Users\<username>\AppData\Local\Continuum\miniconda3"?I cannot find it.@jmarkow

**Comment by yangliu666yl** [comment]
> Hi all, I ran into the same problem and was able to fix it through copying all of dlls generated in the libfreenect2 build (in `build/install/bin` by default in the libfreenect2 source directory),
> 
> ```
> freenect2.dll
> glfw3.dll
> libusb-1.0.dll
> turbojpeg.dll
> ```
> 
> to my miniconda3 path,
> 
> `C:\Users\<username>\AppData\Local\Continuum\miniconda3`
> 
> Alternatively I believe you could append the path to the dlls to your path variable,
> 
> ```
> set path=%PATH%;C:\libfreenect2\build\install\bin
> ```

thanks a lot

**Comment by MuhammadBilal848** [comment]
> ###
> ** [jmarkow](/jmarkow)**

It worked!!!!!!

I found the "F:\vcpkg\installed\x64-windows\bin"  and added it to my PATH.
